examples/iam-identity.v1.test.js

```diff
@@ -2302,6 +2306,132 @@ test('createApiKey request example', async () => {
     }
     // end-delete_all_versions_of_account_settings_template
   }
+  test('updatePreferenceOnScopeAccount request example', async () => {
+
+    consoleLogMock.mockImplementation(output => {
+      originalLog(output);
+    });
+    consoleWarnMock.mockImplementation(output => {
+      originalWarn(output);
+      // when the test fails we need to print out the error message and stop execution right after it
+      expect(true).toBeFalsy();
+    });
+
+    expect(iamIDForPreferences).not.toBeNull();
+
+    originalLog('updatePreferenceOnScopeAccount() result:');
+    // begin-update_prefernce_on_scope_account
+
+    const params = {
+      accountId:    accountId,
+			iamId:        iamIDForPreferences,
+			service:      service,
+			preferenceID: preferenceID1,
+			valueString:  valueString,
+    };
+
+    try {
+      const res = await iamIdentityService.updatePreferenceOnScopeAccount(params);
+      console.log(JSON.stringify(res.result, null, 2));
+    } catch (err) {
+      console.warn(err);
+    }
+
+    // end-update_prefernce_on_scope_account
+  });
+  test('getPreferencesOnScopeAccount request example', async () => {
+
+    consoleLogMock.mockImplementation(output => {
+      originalLog(output);
+    });
+    consoleWarnMock.mockImplementation(output => {
+      originalWarn(output);
+      // when the test fails we need to print out the error message and stop execution right after it
+      expect(true).toBeFalsy();
+    });
+
+    expect(iamIDForPreferences).not.toBeNull();
+
+    originalLog('getPreferencesOnScopeAccount() result:');
+    // begin-get_prefernces_on_scope_account
+
+    const params = {
+      accountId:    accountId,
+			iamId:        iamIDForPreferences,
+			service:      service,
+			preferenceID: preferenceID1,
+    };
+
+    try {
+      const res = await iamIdentityService.getPreferencesOnScopeAccount(params);
+      console.log(JSON.stringify(res.result, null, 2));
+    } catch (err) {
+      console.warn(err);
+    }
+
+    // end-get_prefernces_on_scope_account
+  });
+  test('getAllPreferencesOnScopeAccount request example', async () => {
+
+    consoleLogMock.mockImplementation(output => {
+      originalLog(output);
+    });
+    consoleWarnMock.mockImplementation(output => {
+      originalWarn(output);
+      // when the test fails we need to print out the error message and stop execution right after it
+      expect(true).toBeFalsy();
+    });
+
+    expect(iamIDForPreferences).not.toBeNull();
+
+    originalLog('getAllPreferencesOnScopeAccount() result:');
+    // begin-get_all_prefernces_on_scope_account
```

Comment:
```suggestion
    // begin-get_all_preferences_on_scope_account
```

Reply:
@padamstx all the changes address Phil, hope it is okay now.. we are good to merge ?